install_config/router/default_haproxy_router.adoc

```diff
@@ -1516,11 +1527,19 @@ for the router dc and delete the following lines:
 ----
 ====
 Where 1936 is the STATS_PORT value.
++
+Launch the stats window use the following URL in a browser:
++
+----
+http://admin:<Password>@localhost:1936/metrics
```

Comment:
"/metrics" is not needed here since any path (or no path) will yield the same results. Again, localhost is the IP or FQN of the router.